src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,15 @@
+package racingcar.domain;
+
+public class Car {
+
+    private final int INCREASE_STANDARD_NUMBER = 4;
```

Comment:
```suggestion
    private static final int INCREASE_STANDARD_NUMBER = 4;
```
static이 빠져있다면 Car 객체가 10개가 생성되면 10개의 INCREASE_STANDARD_NUMBER가 생성됩니다.
모든 Car마다 같은 변수를 공유할 수 있도록 static 키워드를 붙여주면 좋지 않을까요?

Reply:
앗 맞습니다. 각각의 Car 객체의 인스턴스 메모리에 저장하기 보다 해당 값은 static으로 선언해서 사용하는 것이 더 메모리 측면에서 효율적이네요!